qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -1523,7 +1531,7 @@
 xxminusyy = QuantumCircuit(q)
 beta = Parameter("beta")
 theta = Parameter("theta")
-rules = [
+rules: list[tuple[Gate, list[Qubit], list[Clbit]]] = [
```

Comment:
This is mainly a question about mypy: here the variable is directly defined, so I would assume the type is known. Is it simply that for complex constructs mypy cannot infer the type, or why did it warn here?

Reply:
Idk, locally I get
```
qiskit/circuit/library/standard_gates/equivalence_library.py:1499: error: Need type annotation for "rules"  [var-annotated]
```